File: 2427-first-letter-to-appear-twice/solution.java
class Solution {
    public char repeatedCharacter(String s) {
       ArrayList<Character> list = new ArrayList <>();
       for(char ch:s.toCharArray()){
            if(list.contains(ch)){
                return ch;
            }
            else{
                list.add(ch);
            }}
    return 0;
    }
}

File: 2427-first-letter-to-appear-twice/solution.c
char repeatedCharacter(char* s) {

    int see[26];
    for(int i = 0 ; i < 26 ;i++){
        see[i]=0;
    }
    int i = 0;
    while (s[i] != '\0'){
        char cu = s[i];
        int index = cu-'a';
        if(see[index]==1)
            return cu;
        see[index] =1;
        i++;
    }
    return '\0';
}
